.github/workflows/links.yml

```
@@ -0,0 +1,22 @@
+name: Links
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.0.9
+        with:
+          args: --verbose --no-progress **/*.md **/*.html
```

```suggestion
          args: --verbose --no-progress --require-https  **/*.md **/*.html
```

I think we should definitely check for https. I know this will cause more errors, but it'll help us catch typos in the future.

---

Yes it will cause more errors and all websites with http will have to be changed to https to pass the test right?

---

Yes, but that's acceptable. We want to verify that the Action will fail and then we apply fixes to it.